cluster-provision/gocli/go.mod

```diff
@@ -19,28 +19,29 @@ require (
 	k8s.io/apimachinery v0.20.6
 )
 
+require (
```

Comment:
@xpivarc why are these not included with the rest of the indrect deps below?

Reply:
Good question. I used standard tooling with no manual changes, therefore I am not sure.

Reply:
Seems to be a common issue - https://github.com/golang/go/issues/56471

Not sure if it is worth manually tidying this?